src/tests/cwrapper/test_cwrapper.c

```diff
@@ -43,6 +43,18 @@ void test_cwrapper() {
     printf("Is_a_Rational %s: %d\n", s, is_a_Rational(e));
     printf("Is_a_Integer %s: %d\n", s, is_a_Integer(e));
 
+    integer_set_ui(e, 123);
+    printf("integer_get_ui 123: %lu\n", integer_get_ui(e));
+
+    integer_set_si(e, -123);
+    printf("integer_get_si -123: %ld\n", integer_get_si(e));
+
+    mpz_t test;
+    mpz_init(test);
```

Comment:
Since you are initializing `test` here, you'll have to clear it with `mpz_clear` after you are done with `test`.